src/map/script.cpp

```diff
@@ -25812,6 +25812,17 @@ BUILDIN_FUNC(randomoptgroup)
 	return SCRIPT_CMD_SUCCESS;
 }
 
+BUILDIN_FUNC( open_quest_ui ){
+	struct map_session_data* sd;
+
+	if (!script_rid2sd(sd))
+		return SCRIPT_CMD_FAILURE;
+
```

Comment:
Maybe add a check here if the quest exists?

Reply:
When the quest doesn't exist (example `open_quest_ui(50)`) the client display

![Capture d’écran (651)](https://user-images.githubusercontent.com/5231893/158022734-95e3c6c4-47c2-457e-9450-517bc0e78583.png)
it doesn't seem necessary to check if the quest exists. Do you want to check anyway?

Reply:
I guess if it's checked with the LUB the client won't crash, which shouldn't matter since this is only to visualize a quest to a player, not activate one.

I say we should be ok without including a quest_db check.

Reply:
Basically it is ok, but could be helpful to get a server side message.
You never know if players report it or not.